### ISSUE TITLE
mpv support now requires compiler and dev libraries

### ISSUE THREAD

**Issue by nbastin** [xgi/castero#19]
There's no wheels for pympv, so installing Castero now requires your system have a compiler and the mpv headers.  Is there any way we can make this a conditional requirement?

**Comment by xgi** [comment]
Yeah, this wasn't intended behavior. python-vlc doesn't require a VLC installation so I wrongly assumed the same would be the case here.

I've added a hacky solution to the install process which simply checks whether the mpv executable is available before installing pympv. At least one concern with this is that if castero is installed before mpv, the client won't have pympv available and so it will need to be reinstalled. I've added a warning about this in some of the relevant error messages, but it should be added to the readme as well for the next release.

Unfortunately distutils doesn't seem to offer a good solution here. I think ideally we would have a set of dependencies (i.e. python-vlc, pympv, etc) where only >= 1 of them needs to be satisfied. `extras` doesn't really seem to solve this, since although it allows for making a dependency optional, I don't want to obfuscate the installation process with additional optional commands. Perhaps in the future we could add a wrapper around the install to let the user make some of these choices. It would also probably be wise to indicate somewhere in the client which player is being used, since it sometimes ignores the config option.
